src/pages/components/input.mdx

```diff
@@ -17,17 +17,21 @@ components:
 <Usage>
   <Use>
     <ul>
-      <li>To let the user enter data</li>
-      <li>To enter multiline text, use a Textarea</li>
+      <li>To let the user enter data into a field</li>
     </ul>
   </Use>
+  <Misuse>
+    <ul>
+      <li>To enter multiline text, use a Textarea</li>
```

Comment:
Just a bit odd to see a "Not for this" and, yet, the `Textarea` component exists on this page.

Reply:
Good callout. This seems more like a when to use

Reply:
@vsri1137 You ok with moving this to When to use?